src/explorer.ts

```diff
@@ -400,6 +401,17 @@ class KubernetesSelectsPodsFolder extends KubernetesResourceFolder {
     }
 }
 
+class KubernetesCRDFolder extends KubernetesFolder {
+    constructor(readonly kind: kuberesources.ResourceKind) {
+        super(kind.abbreviation, kind.pluralDisplayName);
+    }
+
+    async getChildren(kubectl: Kubectl, host: Host): Promise<KubernetesObject[]> {
+        const objects = await kubectlUtils.getCRDTypes(kubectl);
+        return objects.map((obj) => new KubernetesResourceFolder(new kuberesources.ResourceKind(obj.metadata.name, obj.spec.names.plural, obj.kind, obj.spec.names.shortNames !== null ? (obj.spec.names.shortNames.length > 0 ? obj.spec.names.shortNames[0] : "") : "")));
```

Comment:
Are you sure you want `obj.kind` rather than `obj.names.kind`?  Shouldn't the first argument (`displayName`) be `obj.names.singular`?  And for abbreviation (fourth argument) I'd suggest that if `shortNames` is empty then it should be `obj.metadata.name` rather than the empty string?

Reply:
Yeah, I did it this way because I recalled that singular is not required. From what I see `obj.names.singular` is not required **but** it `Defaults to lowercased <kind>` (kubectl explain crd.spec.names.singular), so I'm gonna do it your way, thanks!